test/distributed/checkpoint/test_dedup_tensors.py

```diff
@@ -0,0 +1,45 @@
+# Owner(s): ["oncall: distributed"]
+
+import dataclasses
+import torch
+from torch.distributed.checkpoint.dedup_tensors import dedup_tensors
+from torch.distributed.checkpoint.planner import SavePlan, WriteItemType
+from torch.distributed.checkpoint.planner_helpers import (
+    _create_write_item_for_tensor,
```

Comment:
nit: is there a standard in checkpointing code where public and private APIs are exposed? i.e. for this `planner_helpers` we can also make it a public API since it's being used out of the `planner_helpers` itself?

Reply:
I am actually not 100% sure of what the distinction for public and private APIs for checkpoint. I am following what Rodrigo has previously here. To my observation, it seems everything used by a class is public and every helper function under public function is private.